dftimewolf/lib/collectors/gcloud.py

```diff
@@ -66,7 +66,10 @@ def Process(self):
           disk.name, new_disk.name))
       new_disk.AddLabels(self._gcp_label)
       self.analysis_vm.AttachDisk(new_disk)
-      self.state.output.append((self.analysis_vm.name, new_disk))
+      self.state.StoreContainer(containers.ForensicsVM(
```

Comment:
Please separate the creation of the container from storing it, to make this easier to read and debug